code/lib/eslint-plugin/package.json

```diff
@@ -25,15 +25,22 @@
       "email": "rafaelrozon.developer@gmail.com"
     }
   ],
-  "main": "dist/index.js",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
+    },
+    "./package.json": "./package.json"
+  },
   "files": [
     "dist/**/*",
     "README.md"
   ],
   "scripts": {
     "check": "jiti ../../../scripts/check/check-package.ts",
     "generate-rule": "jiti ./scripts/generate-rule",
-    "prep": "jiti ../../../scripts/prepare/bundle.ts",
+    "prep": "jiti ../../../scripts/build/build-package.ts",
     "update-rules": "jiti ./scripts/update-all.ts"
```

Comment:
I think we can remove these 2 command `generate-rule` and `update-rules`, can't we?

Reply:
we can remove `update-rules` but `generate-rule` is a utility to scaffold new rules when needed, it's unrelated to the build setup